.github/workflows/test_build_conda_linux_without_cuda.yml

```diff
@@ -24,18 +25,18 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - repository: pytorch/text
+          - repository: pytorch/data
```

Comment:
A note from TorchData side, we do have a flag called `BUILD_S3=1` to enable compilation of our C++ code for binary releases.

Reply:
Thanks for such details @ejguan!